tests/js/system/data/acf4.php

```diff
@@ -100,4 +100,42 @@
 		),
 		'menu_order' => 0,
 	));
+
+	register_field_group(array (
```

Comment:
Can you add a space before `array`

Reply:
As this is coming out of the ACF Export function like this I'd rather leave it like this as it makes it easy to just C&P stuff on changes and having clean diffs. What do you think about that?

Reply:
That 's okay for me 👍